packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/Statements/SingleLineCommentStatement.cs

```diff
@@ -5,8 +5,15 @@
 
 namespace Microsoft.Generator.CSharp.Statements
 {
-    public sealed record SingleLineCommentStatement(FormattableString Message) : MethodBodyStatement
+    public sealed class SingleLineCommentStatement : MethodBodyStatement
```

Comment:
well a shortage of this is that
we still cannot put comments in some places, for example our system does not allow we add comments above the `case` in a `switch` like this:
```
switch(something)
{
// case 1
case 1:
break;
}
```

Reply:
Is this a short coming on this statement all together?  If so since my PR is not adding this can you open / link a follow up issue to track that?